activesupport/lib/active_support/testing/event_reporter_assertions.rb

```diff
@@ -0,0 +1,160 @@
+# typed: true
+# frozen_string_literal: true
+
+module ActiveSupport
+  module Testing
+    module EventReporterAssertions
+      module EventCollector # :nodoc:
+        @subscribed = false
+        @mutex = Mutex.new
+
+        class Event
```

Comment:
This class is still showing up as documented, for some reason:
https://shopify-ac-structured-events.rails-docs-preview.pages.dev/api/classes/ActiveSupport/Testing/EventReporterAssertions/EventCollector/Event

Reply:
This is still showing up

Reply:
Arf, thought this would go away with a `# :nodoc: all`. I'll investigate.

Reply:
Okay even with explicit `nodocs` these show up 😕 Not sure what to do there. Seems like we have the same issue with related objects under the `ErrorReporterAssertions` namespace: https://shopify-ac-structured-events.rails-docs-preview.pages.dev/api/classes/ActiveSupport/Testing/ErrorReporterAssertions/ErrorCollector/Report